config/sync/core.entity_view_display.paragraph.os2loop_section_page_views_refer.default.yml

```diff
@@ -3,26 +3,24 @@ langcode: en
 status: true
 dependencies:
   config:
-    - field.field.paragraph.os2loop_section_page_views_refer.os2loop_section_page_view
+    - field.field.paragraph.os2loop_section_page_views_refer.field_os2loop_section_page_block
```

Comment:
Remove the `field_` prefix (cf. https://github.com/itk-dev/os2loop/tree/develop/docs/development#fields).